pkgs/applications/video/anime-downloader/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, python3Packages, aria2, mpv, nodejs }:
+with python3Packages;
+let
+  pySmartDL = buildPythonPackage rec {
```

Comment:
I am not a python maintainer but I think they would prefer to see these as their own packages, to avoid other packages to repackage those.

Reply:
Thanks for the reply! So it would be best if we split these in 3 PRs (anime-downloader; pySmartDL; cfscrape)?

Reply:
that's it, and ideally you put yourself maintainer of those 3 even if in practice people help eachother even when they are not maintainers

Reply:
also with a mention as to why the tests are disabled. I am of the opinion not to overload maintainers when they add the package, and that the test can be added later but not every maintainer agrees and having a small comment can help.

Reply:
> ` results/anime-downloader/bin/anime gui` doesn't start but taht wouldnt be blocking.
> 
> I've first tried
> 
> ```
> results/anime-downloader/bin/anime watch -n
> anime: anime-downloader 5.0.9
> Enter a anime name or url: spy family
> ```
> 
> which blocked then I've tried the example from the tutorial https://anime-downlader.readthedocs.io/en/latest/usage/watch.html#anime-watch-new but for me it looks stuck again
> 
> ```
> results/anime-downloader/bin/anime watch 'code geass' -n
> anime: anime-downloader 5.0.9
> ```
> 
> I wonder if it's due to packaging or a bug in the program ?

This is my config file as an example: https://pastebin.com/raw/A5dqcsK1 (I had trouble with the default provider too)

Reply:
PRs: #176421 and #176416.